handler/src/main/java/io/netty/handler/ssl/JdkSslServerContext.java

```diff
@@ -105,7 +150,34 @@ public JdkSslServerContext(
      * @param sessionCacheSize the size of the cache used for storing SSL session objects.
      *                         {@code 0} to use the default value.
      * @param sessionTimeout the timeout for the cached SSL session objects, in seconds.
-     *                       {@code 0} to use the default value.
+     *                        {@code 0} to use the default value.
+     * @param keyStore the used keystore type
+     * @deprecated use {@link SslContextBuilder}
+     */
+    @Deprecated
+    public JdkSslServerContext(
```

Comment:
It does not make sense to expose a protected method and mark it as @Deprecated from the beginning. Maybe you should remove the annotation and make it package-private ?